lib/ui/home/cards/accounts/detail/coins/coin_tag_details_screen.dart

```diff
@@ -168,22 +168,21 @@ class _CoinTagWidgetState extends ConsumerState<CoinTagDetailsScreen> {
                   fillColor: Colors.transparent,
                   secondaryAnimation: secondaryAnimation,
                   transitionType: SharedAxisTransitionType.scaled,
-                  child: child,
+                  child: Align(alignment: Alignment.topCenter, child: child),
                 );
               },
               child: _selectedCoin != null
-                  ? GestureDetector(
-                      onTap: () {}, // if you tap inside the window do not exit
```

Comment:
Why get rid of `GestureDetector`?

Reply:
Is it not necessary anymore for what's in the comment?

Reply:
this is handled outside of the detail widget. so this one is deadcode